### PR TITLE
chore: rebrand, add lint and CI

### PR DESCRIPTION
**Pull request by AndreyBelym**
* Renamed to be "angular-testcafe"
* License and docs changes due to renaming and the new owner
* Added eslint for the builder
* Added GitHub Actions to run lint and e2e tests on master & PRs

### CODE REVIEW

**Review by kirovboris — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Serves the Angular application, and then runs the TestCafe tests.
 
 ## Install
 ```bash
-$ npm install --save-dev @politie/angular-testcafe-builder
+$ npm install --save-dev angular-testcafe
```
Comment (helen-dikareva) [inline README.md:7]: If we remove travis CI tests let's remove the badge in the title of README

Comment (helen-dikareva) [reply README.md:7]: Should we do something with the warning?
image

Comment (AndreyBelym) [reply README.md:7]: We can. It's just a bad habit to use `any` everywhere, and we try to not overuse it with this rule in Framework

**Review by helen-dikareva — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,10 +1,16 @@
-# angular-testcafe [![Build Status](https://travis-ci.org/politie/angular-testcafe.svg?branch=master)](https://travis-ci.org/politie/angular-testcafe)
+# angular-testcafe [![Tests - E2E](https://github.com/DevExpress/angular-testcafe/actions/workflows/test-e2e.yml/badge.svg)](https://github.com/DevExpress/angular-testcafe/actions/workflows/test-e2e.yml)
+
 A custom Angular builder for [TestCafe](http://devexpress.github.io/testcafe/).
 Serves the Angular application, and then runs the TestCafe tests.
 
+# ⚠️**angular-testcafe supersedes @politie/angular-testcafe-builder**⚠️
+Original [@politie/angular-testcafe-builder](https://www.npmjs.com/package/@politie/angular-testcafe-builder) authors (Dutch National Police and Tim Nederhoff in person) have transfered repository and related code, assets and rights to DevExpress. It means that now it is officially maintained and supported under the name [angular-testcafe](https://www.npmjs.com/package/angular-testcafe) by original TestCafe authors - the TestCafe team.
```
Comment (DIRECTcut) [inline README.md:7]:
```suggestion 
[@politie/angular-testcafe-builder](https://www.npmjs.com/package/@politie/angular-testcafe-builder) authors (Dutch National Police and Tim Nederhoff in person) have transferred the repository, related code, assets, and rights to DevExpress. The TestCafe team maintains and supports the package under the new name [angular-testcafe](https://www.npmjs.com/package/angular-testcafe). 
```

File: README.md [hunk 3]
```diff
@@ -1,10 +1,16 @@
...
+
 A custom Angular builder for [TestCafe](http://devexpress.github.io/testcafe/).
 Serves the Angular application, and then runs the TestCafe tests.
 
+# ⚠️**angular-testcafe supersedes @politie/angular-testcafe-builder**⚠️
+Original [@politie/angular-testcafe-builder](https://www.npmjs.com/package/@politie/angular-testcafe-builder) authors (Dutch National Police and Tim Nederhoff in person) have transfered repository and related code, assets and rights to DevExpress. It means that now it is officially maintained and supported under the name [angular-testcafe](https://www.npmjs.com/package/angular-testcafe) by original TestCafe authors - the TestCafe team. 
+
+If you are still using [@politie/angular-testcafe-builder](https://www.npmjs.com/package/@politie/angular-testcafe-builder), it's strongly advised to switch your project to use [angular-testcafe](https://www.npmjs.com/package/angular-testcafe) instead. The development of [@politie/angular-testcafe-builder](https://www.npmjs.com/package/@politie/angular-testcafe-builder) is discontinued. It won't receive any updates, even for critical bugs and security vulnerabilities.
```
Comment (DIRECTcut) [inline README.md:9]:
```suggestion
If your projects are still using [@politie/angular-testcafe-builder](https://www.npmjs.com/package/@politie/angular-testcafe-builder), we recommend that you switch to [angular-testcafe](https://www.npmjs.com/package/angular-testcafe). The development of [@politie/angular-testcafe-builder](https://www.npmjs.com/package/@politie/angular-testcafe-builder) is discontinued: it won't receive any updates, even for critical bugs and security vulnerabilities.
```

**Review by DIRECTcut — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (timnederhoff) [reply README.md:7]: Additionally, the @politie/angular-testcafe module in npmjs.com can be set as deprecated: https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions and referred to the new version

Comment (AndreyBelym) [reply README.md]: @timnederhoff, thank you so much for everything that you do for the testing community and TestCafe, in particular 👍 We would really like to notify existing users about the new version via `npm deprecate` if you and your former colleagues don't mind it. Unfortunately, we'll have to bother you again to complete this task.

Comment (rkettelerij) [reply README.md]: Once you've cut a new release l, I'll mark the existing package as deprecated in npm.

Comment (AndreyBelym) [reply README.md]: Thank you so much, I just published `angular-testcafe@2.0.6`. After some thinking, I decided to continue original versioning to preserve existing tags and releases in the repository.

Comment (rkettelerij) [reply README.md]: @AndreyBelym @timnederhoff I've deprecated https://www.npmjs.com/package/@politie/angular-testcafe-builder in the NPM registry.

**Pull request merged**